### PR TITLE
BLD/TST: Get the pypi builds running on gha

### PR DESCRIPTION
**Pull request by ZLLentz**
- Update the pyproject.toml to be more like all of our other packages
- Add some really basic tests, no functionality tests yet but checks the very surface-level things

Sneaky extra change:
- Update the base pv for the xrt optics plc (https://github.com/pcdshub/lcls-plc-xrt-optics/pull/48#discussion_r1164416576) (note: the lfe configuration is currently unused as the database is not deployed on lfe)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
+write_to = "pmpsdb_client/_version.py"
+
+[tool.setuptools.packages.find]
+where = [ ".",]
+include = [ "pmpsdb_client*",]
+namespaces = false
+
+[tool.setuptools.dynamic.readme]
+file = "README.md"
+content-type = "text/markdown"
```
Comment (klauer) [inline pyproject.toml:36]: 👍

**Review by klauer — APPROVED** [review]

**Comment by ZLLentz** [comment]
I might push one more commit to nitpick the readme a bit since it's going to become the pypi page

**Review by tangkong — APPROVED** [review]
LGTM

**Comment by ZLLentz** [comment]
One more last commit to delete the travis config and then this is g2g

**Pull request merged**